shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -534,6 +534,11 @@ - (void)updateEditingClient:(int)client withState:(NSDictionary*)state {
                               arguments:@[ @(client), state ]];
 }
 
+- (void)updateEditingClient:(int)client withState:(NSDictionary*)state withTag:(NSString*)tag {
+  [_textInputChannel.get() invokeMethod:@"TextInputClient.updateEditingStateWithTag"
```

Comment:
This would be safer if you used performSelector:withObject:withObject: https://developer.apple.com/documentation/objectivec/1418956-nsobject/1418667-performselector?language=objc

That will give you static analysis that the method you are invoking exists.

Reply:
I think `@"TextInputClient.updateEditingStateWithTag"` does not represent an objective-c selector but a platform message (that will be received and handled by some dart code)?

Reply:
sorry, you're right.